mephisto/abstractions/blueprints/remote_procedure/remote_procedure_agent_state.py

```diff
@@ -101,6 +101,8 @@ def get_data(self) -> Dict[str, Any]:
             "requests": [r.to_dict() for r in self.requests.values()],
             "start_time": self.start_time,
             "end_time": self.end_time,
+            # TODO: Add a metadata field here which handles tips that the agent submits
+            # TODO: Create a script that can review these tips by looping through all agents
```

Comment:
we can remove this TODO. we decided not to include this in `remote_procedure`...